examples/fastatic/index.js

```diff
@@ -3,12 +3,12 @@
 const formatter = require('../../index');
 const path = require('path');
 const fs = require('fs');
+const readme = fs.readFileSync(path.join(__dirname, '/README.md'), 'utf8');
```

Comment:
would be nice if system i/o was async too:

```javascript
fs.readFile(path.join(__dirname, '/README.md'), 'utf8'), readme => {
  formatter(readme)
  //...
});
```

or maybe better yet, promisify all system i/o:
```javascript
const promisifyAll = require('bluebird').promisifyAll;
const fs = promisifyAll(require('fs'));

const srcFilename = path.join(__dirname, '/README.md');
const destFilename = path.join(__dirname, '/index.html');

fs.readFile(srcFilename, 'utf8')
  .then(readme => formatter(readme))
  .then(html => fs.writeFile(destFilename, html))
  .catch(err => console.error(err));
```

This way you also do error handling for read and write as you end with a `catch` which picks up all unhandled rejections.